### PR TITLE
release-4.5: prepare CSV for 4.5.2 z-stream release

### PR DESCRIPTION
**Pull request by fromanirh**
Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

File: deploy/olm-catalog/performance-addon-operator/4.5.2/performance-addon-operator.v4.5.2.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+              "node-role.kubernetes.io/performance": ""
+            },
+            "realTimeKernel": {
+              "enabled": true
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    containerImage: REPLACE_IMAGE
```
Comment (yanirq) [inline deploy/olm-catalog/performance-addon-operator/4.5.2/performance-addon-operator.v4.5.2.clusterserviceversion.yaml:47]: Comparing with 4.5.1 csv I'm missing the following fields here: 
```
    categories: OpenShift Optional
    certified: 'false'
    description: Operator to optimize OpenShift clusters for applications sensitive
      to CPU and network latency.
    repository: https://github.com/openshift-kni/performance-addon-operators
    support: Red Hat
```

File: deploy/olm-catalog/performance-addon-operator/4.5.2/performance-addon-operator.v4.5.2.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    type: AllNamespaces
+  keywords:
+  - numa
+  - realtime
+  - cpu pinning
+  - hugepages
+  links:
+  - name: Source Code
+    url: https://github.com/openshift-kni/performance-addon-operators
+  maturity: alpha
```
Comment (yanirq) [inline deploy/olm-catalog/performance-addon-operator/4.5.2/performance-addon-operator.v4.5.2.clusterserviceversion.yaml:188]: comparing to d/s im missing :
```
  maintainers:
  - email: openshift-operators@redhat.com
    name: Red Hat
```

File: deploy/olm-catalog/performance-addon-operator/4.5.2/performance-addon-operator.v4.5.2.clusterserviceversion.yaml (hunk 1 above)
Comment (fromanirh) [reply deploy/olm-catalog/performance-addon-operator/4.5.2/performance-addon-operator.v4.5.2.clusterserviceversion.yaml:47]: added by `annotations.json`

File: deploy/olm-catalog/performance-addon-operator/4.5.2/performance-addon-operator.v4.5.2.clusterserviceversion.yaml (hunk 2 above)
Comment (fromanirh) [reply deploy/olm-catalog/performance-addon-operator/4.5.2/performance-addon-operator.v4.5.2.clusterserviceversion.yaml:188]: added by `maintainers.json`

**Pull request merged**